definitions/npm/next_v7.x.x/flow_v0.53.x-/next_v7.x.x.js

```diff
@@ -65,21 +65,21 @@ declare module "next/head" {
   declare module.exports: Class<React$Component<any, any>>;
 }
 
-declare module "next/link" {
-  declare export type URLObject = {
-    +href?: string,
-    +protocol?: string,
-    +slashes?: boolean,
-    +auth?: string,
-    +hostname?: string,
-    +port?: string | number,
-    +host?: string,
-    +pathname?: string,
-    +search?: string,
-    +query?: Object,
-    +hash?: string
-  };
+declare type URLObject = {
```

Comment:
Can you talk about why you moved it outside the module definition? Generally we try to keep them contained.

Reply:
Nope that seems reasonable thanks for explaining.